Pages/Index.cshtml

```diff
@@ -69,35 +69,35 @@
 
 
         </ul>
-        <hr> <!--placed horizontal line outside-->
+         <!--placed horizontal line outside-->
```

Comment:
Why was the hr removed, and the comment remains?  This comment is now misleading.